content/docs/introduction/regions.md

```diff
@@ -9,7 +9,7 @@ updatedOn: '2024-09-03T14:54:52.675Z'
 
 Neon supports project deployment in several regions. We recommended that you select the region closest to your application server to reduce latency between your Neon database and your application.
 
-## Available regions
+## AWS regions
 
 Neon currently supports the following AWS regions:
```

Comment:
Should this line be moved outside of AWS regions and made to capture all clouds under `Available regions?`?

Reply:
I've removed this line.